dev/bootstrap.cfg

```diff
@@ -16,6 +16,8 @@ puppetlabs.trapperkeeper.services.metrics.metrics-service/metrics-service
 puppetlabs.trapperkeeper.services.metrics.metrics-service/metrics-webservice
 puppetlabs.services.jruby.jruby-metrics-service/jruby-metrics-service
 puppetlabs.services.analytics.analytics-service/analytics-service
+# To disable automatic CRL reloading, comment the line below.
+puppetlabs.trapperkeeper.services.watcher.filesystem-watch-service/filesystem-watch-service
```

Comment:
This bootstrap.cfg is just used for dev environment purposes. There's a couple other .cfg's that actually go into production. I think we want to add this to the ca.cfg, which ultimately lands in the conf dir we expect users to edit.

Reply:
Perfect. But we should probably also remove the line from the ezbake bootstrap.cfg because both it and the ca.cfg go live in packages, and the resulting service load set is the union of both files - which means as-is they could comment it out in ca.cfg and it would still get loaded.

Reply:
Okay good to know. Do we want to update 2.x to only have it in one spot then too? It's currently commented out in both places.

Reply:
I figured it was fine to leave in 2.x because a user would only actually uncomment it one place (the ca.cfg) but it might be useful to us to know its there / available via the internal bootstrap.cfg ? not sure

Reply:
Works for me. So long as nothing bad happens when it's included twice, that seems fine. If you've uncommented at least one, you're expecting the feature to be enabled.

Reply:
right